### PR TITLE
Log in css

### PR DESCRIPTION
**Pull request by Grub2801**
chnages made based on comments

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -12,7 +12,30 @@
...
 //= require_tree .
 //= require jquery
-//= require bootstrap
\ No newline at end of file
+//= require bootstrap
+
+//Menu Toggle Script
+$(document).ready(function() {
+  if (window.location.pathname == "/"){
+    $("#sidebar-wrapper-right").removeClass("hide");
```
Comment (f3r) [inline app/assets/javascripts/application.js]: saw my comments in the previous pull request? https://github.com/wdi-hk-9/Project-PlaySpace/pull/12#discussion_r43726211

File: app/assets/stylesheets/sidebar.scss [hunk 2]
```diff
@@ -33,7 +33,22 @@
...
+#sidebar-wrapper-right {
+    z-index: 1000;
+    position: fixed;
+    right: 0;
+    width: 0;
+    height: 100%;
+    margin-left: -250px;
+    overflow-y: auto;
+    background: #d8d8d8;
+    -webkit-transition: all 0.5s ease;
```
Comment (f3r) [inline app/assets/stylesheets/sidebar.scss]: https://github.com/wdi-hk-9/Project-PlaySpace/pull/12#discussion_r43725609

**Comment by f3r** [comment]
the rest looks good, merge when fixed the js stuff

**Comment by dshcheung** [comment]
look good

**Pull request merged**